ios/Podfile.lock

```diff
@@ -139,11 +140,16 @@ PODS:
   - nanopb/decode (2.30909.1)
   - nanopb/encode (2.30909.1)
   - OpenSSL-Universal (1.1.1100)
+  - pagopa-io-react-native-cie-pid (0.1.3):
```

Comment:
also podfile should be updated removing `io-react-native-cie-pid`

Reply:
Right! Forgot to run `pod install`

Reply:
Solved in https://github.com/pagopa/io-app/pull/5779/commits/909e6208b32c052bf2469a671f09f268dfd1a6e9